Exiled.Updater/Updater.cs

```diff
@@ -261,88 +292,94 @@ where name.StartsWith("Exiled.", StringComparison.OrdinalIgnoreCase)
 
         #endregion
 
-        private void Update(HttpClient client, NewVersion newVersion)
+        private async Task Update(HttpClient client, NewVersion newVersion)
         {
             try
             {
                 Log.Info("Downloading installer...");
-                using (HttpResponseMessage installer = client.GetAsync(newVersion.Asset.BrowserDownloadUrl).ConfigureAwait(false).GetAwaiter().GetResult())
+                HttpResponseMessage installerResponse = client.GetAsync(newVersion.Asset.BrowserDownloadUrl).ConfigureAwait(false).GetAwaiter().GetResult();
```

Comment:
I didn't find how to make this one await without error

Reply:
ok but HttpResponseMessage is an IDisposable so where's my using directive so it's disposed properly?

And what was the error when using a normal await?